roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -53,13 +66,13 @@
 
     "network_device": "field:port:device_owner=~^network:",
     "create_port": "",
-    "create_port:device_owner": "not rule:network_device or rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:mac_address": "rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:fixed_ips": "rule:admin_or_network_owner or rule:context_is_advsvc",
-    "create_port:port_security_enabled": "rule:admin_or_network_owner or rule:context_is_advsvc",
+    "create_port:device_owner": "not rule:network_device or rule:context_is_advsvc or rule:admin_or_network_owner",
```

Comment:
Yes, it's just an ordering swap. This is how it's listed in the community version now, so I updated it here.